cni/pkg/ebpf/server/redirectionServer_linux.go

```diff
@@ -89,8 +92,132 @@ var stringToLevel = map[string]uint32{
 	"none":  EBPFLogLevelNone,
 }
 
+type eBPFObjects interface {
```

Comment:
I am not sure we need this complexity? Why not just use a standard struct. You can have a special struct for passing to `loadAmbient_redirectObjects`, but then just store them inside a single struct. they are all `*ebpf.Program` so they don't have to be the same.

Basically only the code doing `loadAmbient_redirectObjects` should have to deal with the split path

Reply:
yes, good suggestion. I probably got your point. Simplified and updated it without introducing much interfaces